components/Data.vue

```diff
@@ -126,198 +128,214 @@
+      getCsvIndex: state => state.dataset.csvIndex,
+      loadCsvProgress: state => state.dataset.loadCsvProgress,
+      geojsonError: state => state.dataset.geojsonError,
+      getGeoId: state => state.dataset.geoId,
+      getGeoIndex: state => state.dataset.geoIndex,
+      getGeojsonFiles: state => state.dataset.geojsonFiles,
+      geoProperties: state => state.dataset.geoProperties,
+      loadGeoProgress: state => state.dataset.loadGeoProgress,
+      topojsonError: state => state.dataset.topojsonError,
+      getTopojsonFiles: state => state.dataset.topojsonFiles,
+      getPreLookupAggregate: state => state.dataset.preLookupAggregate,
+      getMode: state => state.dataset.mode,
+      syncError: state => state.syncError,
+    }),
     csvFiles() {
-      return this.$store.state.dataset.csvFiles.map((x, i) => {
+      return this.getCsvFiles.map((x, i) => {
         return { ...x, index: i }
       })
     },
-    availableAggregates() {
-      return Object.keys(aggregateOps)
-    },
-    preLookupAgregate: {
-      get() {
-        return this.$store.state.dataset.preLookupAgregate
-      },
-      set(value) {
-        this.$store.commit('dataset/setPrelookupAgregate', value)
-      },
-    },
     topojsonFiles() {
-      return this.$store.state.dataset.topojsonFiles.map((x, i) => {
+      return this.getTopojsonFiles.map((x, i) => {
         return { ...x, index: i }
       })
     },
     geojsonFiles() {
-      return this.$store.state.dataset.geojsonFiles.map((x, i) => {
+      return this.getGeojsonFiles.map((x, i) => {
         return { ...x, index: i }
       })
     },
-    loadCsvProgress() {
-      return this.$store.state.dataset.loadCsvProgress
+    availableAggregates() {
+      return Object.keys(aggregateOps)
     },
-    loadGeoProgress() {
-      return this.$store.state.dataset.loadGeoProgress
+    preLookupAggregate: {
+      get() {
+        return this.getPreLookupAggregate
+      },
+      set(value) {
+        this.setPreLookupAggregate(value)
+      },
     },
     availableModes() {
-      const modes = []
+      const aModes = []
       if (this.csvFiles.length > 0) {
-        modes.push('csv')
+        aModes.push(modes.csv)
       }
       if (this.topojsonFiles.length > 0) {
-        modes.push('topojson')
+        aModes.push(modes.topojson)
       }
       if (this.geojsonFiles.length > 0) {
-        modes.push('geojson')
+        aModes.push(modes.geojson)
       }
-      if (this.csvFiles.length > 0 && this.topojsonFiles.length > 0) {
-        modes.push('csv + topojson')
-      }
-      if (this.csvFiles.length > 0 && this.geojsonFiles.length > 0) {
-        modes.push('csv + geojson')
-      }
-      return modes
+      // if (this.csvFiles.length > 0 && this.topojsonFiles.length > 0) {
+      //   aModes.push(modes.csvTopojson)
+      // }
+      // if (this.csvFiles.length > 0 && this.geojsonFiles.length > 0) {
+      //   aModes.push(modes.csvGeojson)
+      // }
```

Comment:
Did you mean to keep this?

Reply:
Yes this is how I've disabled the combined modes for now